components/benefit_sponsors/app/views/benefit_sponsors/profiles/employers/employer_profiles/my_account/_benefits.html.erb

```diff
@@ -16,7 +16,7 @@
         <% end %>
       <% end %>
       <%#= pundit_span EmployerProfile,:updateable? %>
-        <% if @benefit_applications.present? && @benefit_applications.published.blank? && !@benefit_applications.any?{|app| app.is_renewing?} %>
+        <% if @benefit_applications.present? && (@benefit_applications.published.blank? || (@benefit_applications.active.present? && @benefit_applications.canceled.select{ |ba| ba.start_on > @benefit_applications.active.first.end_on }.present?)) && !@benefit_applications.any?{|app| app.is_renewing?} %>
```

Comment:
can you clarify, why we needed canceled applications check here...
canceled applications treated as invalid applications and they should not impact AddPlanYear button display.

Reply:
got response from varun, this is to address scenarios where renewal application was canceled and they want to create new one manually as initial with later date. ex. during last month of active application